client/src/components/pages/home/TourneyCard.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+
+import { Button, Typography } from "antd";
+import "./TourneyCard.css";
+
+function redirectToTourney(code) {
+  window.location.href = `/${code}/home`;
+}
+
+export default function TourneyCard({
+  divisions,
+  code,
+  title,
+  description,
+  banner,
+  fullTitle,
+  ongoing,
+}) {
+  // No tournament has divisions for now
+  // const hasDivisions = !!divisions;
+
+  var ongoingClassname = ongoing ? " ongoing" : "";
+
+  return (
+    <div
+      to={`/${code}/home`}
+      className="u-rounded-border TourneyCard-card"
+      style={{
+        backgroundImage: `url("${banner}"), linear-gradient(var(--onyx-dark), var(--onyx-dark))`,
+        backgroundBlendMode: "color",
+        backgroundPosition: "center",
+        backgroundSize: "cover",
+      }}
+      onClick={() => redirectToTourney(code)}
```

Comment:
seems kinda weird to me that clicking anywhere in the div will redirect to the tourney page, esp since there's no cursor pointer and there's a separate learn more button.

personally i'd remove this `onClick`, and add the `onClick` and `cursor: "pointer"` to just the banner div